data/src/test/java/com/linkedin/data/schema/TestPdlBuilder.java

```diff
@@ -0,0 +1,41 @@
+package com.linkedin.data.schema;
+
+import com.linkedin.data.DataMap;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.Collections;
+import java.util.Map;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+
+public class TestPdlBuilder {
```

Comment:
Please use the Rest.li brace style convention (braces on new lines). This goes especially for your data provider, the return value should be expanded neatly.

Reply:
done